### DISCUSSION TITLE
Proposing Armando Ruocco as a new maintainer

### DISCUSSION THREAD

**Discussion by gbartolini** [cloudnative-pg/cloudnative-pg#1023] [Maintainers room]
I am proposing Armando Ruocco as a new CloudNativePG maintainer, considering his contributions to the project so far.

**Comment by mnencia** [comment]
I second the proposal

**Comment by phisco** [comment]
Totally agree :tada:

**Comment by sxd** [comment]
Totally agree!

**Comment by leonardoce** [comment]
I'd be really happy to have Armando as a maintainer!

**Comment by fcanovai** [comment]
I agree

**Reply by gbartolini** [reply]
Approved

**Comment by gbartolini** [comment]
Please all approve this PR: https://github.com/cloudnative-pg/cloudnative-pg/pull/1167
